### PR TITLE
Update image tags to GA versions supplied by ooyala

### PR DESCRIPTION
**Pull request by jemrayfieldft**
Update image tags to GA versions supplied by ooyala
Changed the docker-compose yml stuff.

### CODE REVIEW

File: ansible/templates/docker-compose-components/master-10-master.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
   master:
-    image: registry.ooflex.net/flex-master
+    image: registry.ooflex.net/flex-master:5.3.1.22
```
Comment (jspc) [inline ansible/templates/docker-compose-components/master-10-master.yml]: Needs to be `5.3.1.28`, based on a conversation I just had

File: ansible/cluster/software.yml [hunk 2]
```diff
@@ -131,7 +131,7 @@
     - name: hard lock docker-py for ansible bug
       pip:
         name: docker-py
-        version: 1.9.0
+        version: 1.10.3
```
Comment (jspc) [inline ansible/cluster/software.yml]: How did you even test this? `jemdev` is too old to have been provisioned with just this version. Indeed, new instances still suffer from the bug.

**Comment by jspc** [comment]
@jemrayfieldft, another updated version from Nick.

> Can you fix the version of flex-metadatamerge-service to 1.0.142

File: ansible/cluster/software.yml (hunk 2 above)
Comment (jemrayfieldft) [inline ansible/cluster/software.yml]: will update to 1.10.4

File: ansible/templates/docker-compose-components/master-10-master.yml (hunk 1 above)
Comment (jemrayfieldft) [inline ansible/templates/docker-compose-components/master-10-master.yml]: will update....grr pooyala

**Comment by jspc** [comment]
@jemrayfieldft did you edit my comment? And we should be, if we're not. I had it on `jspcdev` but that got removed in the purge.

**Comment by jemrayfieldft** [comment]
Closing PR. If I had access. Please grant.
Will open more branches and PRs

**Pull request closed without merging**